app/services/childCRUD.ts

```diff
@@ -560,7 +565,8 @@ export const updateActiveChild = (child: any, key: any, value: any, dispatch: an
 }
 export const getAllConfigData = async (dispatch: any): Promise<any> => {
   const allJsonDatanew = await dataRealmCommon.getData<ConfigSettingsEntity>(ConfigSettingsSchema);
-  allJsonDatanew.removeAllListeners();
+  console.log("=====2",allJsonDatanew)
```

Comment:
Make sure these kind of logs do not go to production.

Reply:
done